test/api/package/get-supporting-companies.test.js

```diff
@@ -242,6 +307,8 @@ test('GET `/package/get-supporting-companies` 200 | sorts and only returns top 1
   })
   // Make sure that an undefined org isn't included in the results of this endpoint
   t.false(!!JSON.parse(res.payload).companies.find((c) => c.organizationId === 'aaaaaaaaaaaa'))
+  // Make sure that the org that isnt publically giving is not returned
+  t.false(!!JSON.parse(res.payload).companies.find((c) => c.organizationId === t.context.orgId4))
```

Comment:
rather than parsing the payload twice and asserting that `find` returns something falsy, you could do something like

```suggestion
  const { companies } = JSON.parse(res.payload)
  t.true(companies.every(({ organizationId }) => organizationId !== 'aaaaaaaaaaaa'))
  t.true(companies.every(({ organizationId }) => organizationId !== t.context.orgId4))
```

or something like that, some array method that is designed to state something about all the elements like you're doing